opencog/cython/opencog/atom.pyx

```diff
@@ -10,7 +10,7 @@ cdef class Atom(object):
         del self.handle
 
     def value(self):
-        return void_from_cptr(self.handle)
+        return self.handle.value()
```

Comment:
void_from_cptr(self.handle) doesn't uniquely identify an atom, which is the issue I am trying to fix. I should have mentioned it on the description.
```python
>>> from opencog.atomspace import *
>>> a = AtomSpace()
>>> n1 = a.add_node(types.Node, "foo")
>>> n2 = a.add_node(types.Node, "foo")
>>> n2.value()
30170656
>>> n1.value()
30171824
```